index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset='utf-8'>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv='X-UA-Compatible' content='IE=edge'>
+    <title>Page Title</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round:ital@0;1&family=Inter:wght@700&family=Poppins:ital,wght@0,400;0,700;1,400;1,700&family=Roboto:ital,wght@0,400;0,700;1,400;1,700&display=swap" rel="stylesheet">
+    <link rel='stylesheet' type='text/css' media='screen' href='css/main.css'>
+</head>
+
+<body>
+    <header class="row nav-bar">
+        <a href="/" class="cell"><div class="title">Juan Luis Palacios Perez</div></a>
```

Comment:
- [ ] Well done making the logo a link, but let's fix the syntax. Take into count that any `<div>` tag is always a container, so it cannot be inside of a "non-container" tag. So, please change this line. All you need to do is change the order of the elements like: `<div><a>Your link</a></div>` so this way we are more semantic and we are building our code in the correct manner.